src/main.rs

```diff
@@ -94,17 +96,30 @@ fn get_gps(config: Rc<Config>) -> Box<GPS> {
         }
     }
 
+    /* FIXME: the discovery part should be separated from the RS232 module so that adding
+     * more devices doesn't get even more convoluted.
+     */
```

Comment:
In Rust, you want to use `//` comments.

```suggestion
    // FIXME: the discovery part should be separated from the RS232 module so that adding
    // more devices doesn't get even more convoluted.
```

Reply:
I'll try to take care of the FIXME after this PR is merged (with the suggested change above). I'll continue from there with a new PR. I'm just learning Rust, but this seems to be an easy enough exercise to get some coding experience in Rust :-)

Reply:
@Kareema0 that'd be great. As I say to every single person new to Rust, please do read [this blog post](https://ferrous-systems.com/blog/mindsets-and-expectations/).